ts/features/bonus/cgn/screens/CgnDetailScreen.tsx

```diff
@@ -123,15 +129,18 @@ const CgnDetailScreen = (props: Props): React.ReactElement => {
               )}
             </View>
           </ScrollView>
-          <FooterWithButtons
-            type={"SingleButton"}
-            leftButton={confirmButtonProps(
-              props.isMerchantV2Enabled
-                ? props.navigateToMerchantsTabs
-                : props.navigateToMerchantsList,
-              I18n.t("bonus.cgn.detail.cta.buyers")
-            )}
-          />
+          <SectionStatusComponent sectionKey={"cgn"} />
+          {props.isCgnEnabled && (
+            <FooterWithButtons
+              type={"SingleButton"}
+              leftButton={confirmButtonProps(
+                props.isMerchantV2Enabled
+                  ? props.navigateToMerchantsTabs
+                  : props.navigateToMerchantsList,
```

Comment:
In this case, using a remote feature flag (`isMerchantV2Enabled`) to protect an incomplete functionality, could lead to the activation of an incomplete functionality in this app version, when will be activated:
<img width="300" alt="Schermata 2022-01-13 alle 17 28 22" src="https://user-images.githubusercontent.com/26501317/149369506-1814cc4c-91ec-4c33-be26-fed8a003179c.png">
Is this functionality expected to complete before the app is released? Otherwise I think could be better using a static feature flag until it is completed.

Reply:
81e88c0 adds a proper local FF for the job, thx to catch!